src/abConfiguration.ts

```diff
@@ -42,7 +55,7 @@ class ABConfiguration {
 
     if (splitVariants) {
       const trueVariant = this._getTrueVariant();
-      const trueVariantIndex = splitVariants.indexOf(trueVariant);
+      const trueVariantIndex = splitVariants.indexOf(trueVariant.toString());
```

Comment:
`splitVariants` is an array of strings, so we need to ensure `trueVariant` is a string.